packages/devtools/src/signing.ts

```diff
@@ -0,0 +1,20 @@
+import type { StreamID } from '@ceramicnetwork/streamid'
+import type { DagJWSResult, DID } from 'dids'
+
+import { streamIDToString } from './utils'
+
+export async function signTile<T = unknown>(
+  did: DID,
+  data: T,
+  schema?: StreamID | string
+): Promise<DagJWSResult> {
+  if (!did.authenticated) {
+    throw new Error('DID must be authenticated')
+  }
+
+  const header = {
+    controllers: [did.id],
+    schema: schema ? streamIDToString(schema) : undefined,
+  }
+  return await did.createDagJWS({ data, doctype: 'tile', header }, { did: did.id })
+}
```

Comment:
Are you creating a genesis record here? Why not just use [`TileDocument.makeGenesis()`](https://developers.ceramic.network/reference/typescript/classes/_ceramicnetwork_stream_tile.tiledocument-1.html#makegenesis)?

Reply:
This was just copied over from idx-tools, I don't think it's even needed at the moment so I'll simply remove it.